components/outer-layout/server.js

```diff
@@ -74,7 +74,7 @@ export class OuterLayout extends ServerComponent {
       <!doctype html>
       <html
         lang=${context.locale}
-        style="color-scheme: light dark;"
+        data-theme="light dark"
```

Comment:
Note: We're already using [`data-theme`](https://github.com/search?q=repo%3Amdn%2Ffred%20%2Fdata-theme%2F&type=code) in other places. Are we sure this causes no conflicts?